### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by Fivell**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -4,7 +4,6 @@ env:
     - RAILS=5.2.4.6
     - RAILS=6.0.3.7
 rvm:
-  - 2.4.10
```
Comment (workgena) [inline .travis.yml:7]: Can be removed:
```yml
exclude:
    - rvm: 2.4.10
      env: RAILS=6.0.3.7
```

Comment (Fivell) [reply .travis.yml:7]: @workgena  👍

**Review by workgena — APPROVED** [review]

**Comment by workgena** [comment]
Let's also update CHANGELOG.md and mention that we no longer "officially" support Rails 4, Ruby 2.3

**Review by workgena — APPROVED** [review]
🚀

**Pull request merged**